libraries/bxdf/gltf_pbr.mtlx

```diff
@@ -19,6 +19,7 @@
     <input name="clearcoat_roughness" type="float" value="0" uimin="0" uimax="1" uiname="Clearcoat Roughness" uifolder="Clearcoat" />
     <input name="clearcoat_normal" type="vector3" defaultgeomprop="Nworld" uiname="Clearcoat Normal" uifolder="Clearcoat" />
     <input name="emissive" type="color3" value="0, 0, 0" uimin="0, 0, 0" uimax="1, 1, 1" uiname="Emissive" uifolder="Emission" />
+    <input name="emissive_strength" uniform="true" type="float" value="1" uimin="0" uiname="Emissive Strength" uifolder="Emission" />
     <input name="thickness" uniform="false" type="float" uimin="0" uiname="Thickness" uifolder="Volume" />
```

Comment:
Hi @proog128, a minor thing not directly related. Both `thickness` and `attenutation_distance` have no default values so perhaps worth fixing up with this check-in ? Works for the example since it's explicitly specified in the instance of the node, otherwise not sure what value is used.

Reply:
That's a good question, and I don't believe we yet handle `inf` as a valid scalar value in MaterialX, though this seems like a natural feature to add in the future.  For now, I think you're right to omit this from the shading model declaration, and we can discuss how to handle this feature going forward.